guides/cheatsheets/git.md

```diff
@@ -19,265 +23,256 @@ Version control is the essence of engineering here at the Shed. Git is a free ve
-```
-
-
-*Switch to another branch and check it out into your working directory*.
-```
-git checkout
-```
-
-*Switch to the specified branch. Creates the branch in your local repository if it doesn't already exist*.
-```
-git checkout -b [branch_name]
-```
-
-*Merge the specified branch’s history into the current one*.
-```
-git merge [branch]
-```
-
-*Show all commits in the current branch’s history*.
-```
-git log
-```
-----------
+_List your branches. a\*will appear next to the currently active branch\._
+
+    git branch
+
+_Create a new branch_.
+
+    git branch [branch-name]
+
+_Switch to another branch and check it out into your working directory_.
+
+    git checkout
+
+_Switch to the specified branch. Creates the branch in your local repository if
+it doesn't already exist_.
+
+    git checkout -b [branch_name]
+
+_Merge the specified branch’s history into the current one_.
+
+    git merge [branch]
+
+_Show all commits in the current branch’s history_.
+
+    git log
+
+---
+
 ## **INSPECT & COMPARE**
-----------
+
+---
+
 Examining logs, diffs and object information.
 
-*Show the commit history for the currently active branch*.
-```
-git log 
-```
-
-*Show the commits on branchA that are not on branchB*.
-```
-git log branchB..branchA
-```
-
-*Show the commits that changed file, even across renames*.
-```
-git log --follow [file]
-```
-
-*Get the author of a commit*.
-```
-git blame [commit_hash]
-```
-
-*Show the diff of what is in branchA that is not in branchB*.
-```
-git diff branchB...branchA
-```
-
-*Show any object in Git in human-readable format*.
-```
-git show [SHA]
-```
-----------
+_Show the commit history for the currently active branch_.
+
+    git log
+
+_Show the commits on branchA that are not on branchB_.
+
+    git log branchB..branchA
+
+_Show the commits that changed file, even across renames_.
+
+    git log --follow [file]
+
+_Get the author of a commit_.
+
+    git blame [commit_hash]
+
+_Show the diff of what is in branchA that is not in branchB_.
+
+    git diff branchB...branchA
+
+_Show any object in Git in human-readable format_.
+
+    git show [SHA]
+
+---
+
 ## **TRACKING PATH CHANGES**
-----------
+
+---
+
 Versioning file removes and path changes.
 
-*Delete the file from project and stage the removal for commit*.
-```
-git rm [file]
-```
+_Delete the file from project and stage the removal for commit_.
+
+    git rm [file]
 
-*Change an existing file path and stage the move*.
-```
-git mv [existing-path] [new-path]
-```
+_Change an existing file path and stage the move_.
 
-*Show all commit logs with indication of any paths that moved*.
-```
-git log --stat -M
-```
+    git mv [existing-path] [new-path]
+
+_Show all commit logs with indication of any paths that moved_.
+
+    git log --stat -M
+
+---
 
-----------
 ## **IGNORING PATTERNS**
-----------
+
+---
+
 Preventing unintentional staging or commiting of files.
 
+_Save a file with desired paterns as .gitignore with either direct string
```

Comment:
patterns*

Reply:
Noted/Completed ☑️

Reply:
Should be using the Cookie Cutter or https://www.toptal.com/developers/gitignore  for generating gitignore files?

Reply:
I believe https://www.toptal.com/developers/gitignore is mentioned c. line 248 and since this is a cheatsheet cookie-cutter might be a bit too technology specific? 🥸